src/partials/maintenance-page.hbs

```diff
@@ -0,0 +1,180 @@
+                                    stroke-dasharray="135,1,1,18"
+                                    transform="translate(64.000000, 64.000000) rotate(230.000000) translate(-64.000000, -64.000000) "
+                                    cx="64" cy="64" r="45"></circle>
+                                <circle vector-effect="non-scaling-stroke" class="slds-illustration__stroke-secondary"
+                                    stroke-width="3" stroke-linecap="round" stroke-linejoin="round"
+                                    stroke-dasharray="107,10"
+                                    transform="translate(64.000000, 64.000000) rotate(150.000000) translate(-64.000000, -64.000000) "
+                                    cx="64" cy="64" r="33"></circle>
+                            </g>
+                            <g transform="translate(41.000000, 41.000000)" class="slds-illustration__stroke-primary"
+                                stroke-width="3">
+                                <circle vector-effect="non-scaling-stroke" cx="23" cy="23" r="23"></circle>
+                            </g>
+                        </g>
+                        <g transform="translate(163.000000, 244.000000)" fill="#FFFFFF"
+                            class="slds-illustration__stroke-primary" stroke-linecap="round" stroke-linejoin="round"
+                            stroke-width="3">
+                            <g>
+                                <path vector-effect="non-scaling-stroke"
+                                    d="M0,61 L48,11 C54.6666667,4 63.6666667,0.5 75,0.5 C86.3333333,0.5 113.666667,0.5 157,0.5 C167,1.5 172,6.66666667 172,16 C171,25 162,30.5 157,30.5 C153.666667,30.5 139,30.5 113,30.5 C104.588216,30.8886719 99.8562826,34.3914388 98.8041992,41.0083008 C98.3886719,48.4902344 104.350586,53.5 110,53.5 L198,53.5 C207.225586,53.5 215.892253,49.6666667 224,42 C236.161621,30.5 254.517577,12.615436 263,8 C271.482423,3.384564 284,-5.90638649e-14 293,5 C302,10 303,13 289,26 C279.666667,34.6666667 261.666667,52 235,78 C226.333333,87 215.666667,91.5 203,91.5 C190.333333,91.5 155,91.5 97,91.5 C88.3333333,91.1666667 81,94.6666667 75,102">
+                                </path>
+                            </g>
+                        </g>
+                    </g>
+                </g>
+            </g>
+        </svg>
+        <div class="slds-text-title">
+            <p>docs.mulesoft.com is temporarily down for maintenance</p>
+        </div>
+        {{#if (and page.componentVersion.asciidoc.attributes.maintenanceTime page.componentVersion.asciidoc.attributes.maintenanceDate)}}
+        <div class="slds-text-description">
+            <p>docs.mulesoft.com will be available after <strong>{{page.componentVersion.asciidoc.attributes.maintenanceTime}}</strong> on <strong>{{page.componentVersion.asciidoc.attributes.maintenanceDate}}</strong>. Thank you for your patience as we make important updates to our site.</p>
```

Comment:
instead of `page.componentVersion.asciidoc.attributes.maintenanceTime`, you can specify `asciidoc.attributes.maintenanceTime. Makes it simpler and it is no longer dependent on the page component version.

```
        {{#if (and asciidoc.attributes.maintenanceTime asciidoc.attributes.maintenanceDate)}}
        <div class="slds-text-description">
            <p>docs.mulesoft.com will be available after <strong>{{asciidoc.attributes.maintenanceTime}}</strong> on <strong>{{asciidoc.attributes.maintenanceDate}}</strong>. Thank you for your patience as we make important updates to our site.</p>
        </div>
        {{/if}}
```

Then you can add the maintenance date and time attributes to ui-model.yml. Example:

```yaml
asciidoc:
  extensions:
    - '@asciidoctor/tabs'
  attributes:
    customer-feedback-survey: https://s2.userzoom.com/m/MSBDNTQxOFMxMzI4
    maintenanceTime: '9:00 AM PDT'
    maintenanceDate: 'May 31, 2023'
```